packages/plugin-list/src/ListItemXmlDomParser.ts

```diff
@@ -51,16 +51,10 @@ export class ListItemXmlDomParser extends AbstractParser<Node> {
                 } else {
                     inlinesContainer = null; // Close the inlinesContainer.
                     for (const child of parsedChild) {
-                        const attributes = child.modifiers.find(Attributes);
-                        if (attributes) {
-                            child.modifiers.replace(Attributes, attributes.clone());
-                        }
-                        if (itemAttributes) {
-                            child.modifiers.replace(
-                                ListItemAttributes,
-                                new ListItemAttributes(itemAttributes),
-                            );
-                        }
+                        child.modifiers.replace(
+                            ListItemAttributes,
+                            new ListItemAttributes(itemAttributes),
+                        );
```

Comment:
A part of the previous code was lost here. Not sure if it was intended. Well, now that I read it properly, it seems the old code was useless, wasn't it ? :p

Reply:
Yes I also laughed when I found out :D